operator/v1/types_ingress.go

```diff
@@ -60,6 +60,11 @@ type IngressControllerSpec struct {
 	// +optional
 	Domain string `json:"domain,omitempty"`
 
+	//This used to sync the configmap having customizable error code page created by cluster admin in openshift-config namespace.
+	//to openshift-ingress which will be used by the router to serve appropriate error code pages.
+	//If this field is empty, the default values i.e default shipped error code pages will be served by the haproxy router.
```

Comment:
Some suggestions:
1. Add a space after `//` on each line.  
2. Start with the field's name.  
3. Focus on the usage rather than the implementation.  
4. Describe the content of the configmap (fill in the "[...]" in the suggestion below).  
5. Make the field optional.  
6. Should the field name be plural ("pages" rather than "page")?  
```suggestion
	// httpErrorCodePage specifies a configmap with customizable error pages in the openshift-config namespace.
	// This configmap should have keys [...].
	// If this field is empty, the ingress controller uses the default error pages.
	//
	// +optional
```